packages/module/src/Feedback/FeedbackError.tsx

```diff
@@ -13,7 +13,7 @@ const FeedbackError = ({ onCloseModal }: FeedbackErrorProps) => {
   const intl = React.useContext(LocaleContext);
   return (
     <div className="chr-c-feedback-success-content">
-      <CheckIcon size="md" color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
+      <CheckIcon size={12} color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
```

Comment:
I think that size has been removed entirely, and that you need to replace this with width and height props (of which I think md corresponds with 1.5em IIRC), but I could be a bit mistaken there.

Reply:
Okay I think I can just remove it.  It looks like the default size is fine.